test/steps/3_then.js

```diff
@@ -27,6 +28,27 @@ import {
 	getFirstBoolean,
 } from './utils';
 
+export function itShouldHaveCreatedARegisterSecondPassphraseTransactionUsingThePassphraseAndTheSecondPassphrase() {
+	const { passphrase, secondPassphrase } = this.test.ctx;
+	(createLiskTransaction.createSignature).should.be.calledWith(passphrase, secondPassphrase);
+}
+
+export function itShouldResolveToTheCreatedTransaction() {
+	const { returnValue, createdTransaction } = this.test.ctx;
+	(returnValue).should.be.fulfilledWith(createdTransaction);
```

Comment:
This is secretly async so `return` is actually critical here to avoid a race condition.